dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -1,76 +1,75 @@
 <template>
-  <AuthenticationWrapper>
```

Comment:
Don't we need this Auth-Wrapper any longer?

Reply:
No.  I took it out from here and instead, wrapped it around the parend "ViewFrameworkData.vue".
So its's just moving it up the chain so to say.

Reply:
ok